src/site/_collections/algolia.js

```diff
@@ -0,0 +1,185 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
```

Comment:
I think we should put a big comment in this file explaining how content is organized for algolia and how we search against it.

It sounds like the shape of the data is like this:

```js
{
  title: 'hello',
  locales: ['en']
}, {
  title: 'bon jour',
  locales: ['fr'],
  default_title: 'hello'
}
```

I think it'd be good to have this code snippet in the comment so folks can tell at a glance what's being indexed, and then explain how we search on all of the fields and have english fallback fields in case someone has a language set but is still searching in english